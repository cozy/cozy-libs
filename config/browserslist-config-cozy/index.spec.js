describe('browserslist', () => {
  it('should contain all Cozy supported browsers', () => {
    expect(require('./')).toMatchSnapshot()
  })
})
