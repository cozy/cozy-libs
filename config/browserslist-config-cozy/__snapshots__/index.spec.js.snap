// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`browserslist should contain all Cozy supported browsers 1`] = `
Array [
  "last 2 Chrome major versions",
  "last 2 Firefox major versions",
  "last 2 FirefoxAndroid major versions",
  "last 2 Safari major versions",
  "last 3 iOS major versions",
  "last 2 Edge major versions",
  "Firefox ESR",
  "Android 4.4",
  "> 1% in FR",
  "not dead",
  "not ie <= 11",
]
`;
